airbyte-integrations/connectors/source-google-mobility-reports/source_google_mobility_reports/google_mobility_reports.yaml

```diff
@@ -0,0 +1,32 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: []
+  requester:
+    url_base: "https://www.gstatic.com/covid19/mobility/"
+    http_method: "GET"
+  retriever:
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      type: NoPagination
+    requester:
+      $ref: "*ref(definitions.requester)"
+  base_stream:
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+  global_mobility_report_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "global_mobility_report"
+      primary_key: ["placeid","date"]
+      path: "/Global_Mobility_Report.csv"
```

Comment:
Not sure if this connector should be build using the low-code.

Reply:
@marcosmarxm its a single data-set, so i opted for a simple single stream of data versus providing the filtering. the other filtered data is in PDF and not CSV